src/assertions/defines.h

```diff
@@ -12,31 +12,55 @@
 
 #define GEN_CHECK_EX(check_var, expected_var, varname, func_line) \
     if(check_var != expected_var) { \
-        print( \
-            "  ERROR(line %d): Expected %s = 0x%x, Got = 0x%x", \
-             func_line, varname, expected_var, check_var \
-        ); \
+        if (sizeof(check_var) > 4) { \
```

Comment:
Comparable code appears a few more times below; would it make sense to use a separate define for this, and use that in all applicable places? (The slight difference in the repeated code blocks below seems to be a label that could be passed into the separate define as an argument)

Reply:
If you have suggestion on the how to do it. Feel free to tell me, otherwise I do not know what other alternative method in C standard language.